test_runner/regress/test_timeline_detach_ancestor.py

```diff
@@ -173,9 +196,73 @@ def test_ancestor_detach_branched_from(
     # but if nothing was copied, then there is no nice rule.
     # there could be a hole in LSNs between copied from the "old main" and the first branch layer.
 
+    # take this on the detached, at same lsn
+    fullbackup_after = test_output_dir / "fullbackup-after.tar"
+    cmd = [
+        "psql",
+        "--no-psqlrc",
+        env.pageserver.connstr(),
+        "-c",
+        f"fullbackup {env.initial_tenant} {timeline_id} {branch_at}",
+        "-o",
+        str(fullbackup_after),
+    ]
+    pg_bin.run_capture(cmd, env=psql_env)
+
```

Comment:
I monkey'd this from `test_import.py`. I guess it could be made into reusable fixture, I would prefer to leave it as a follow-up perhaps..

It would be *nicer* to get a fullbackup hash from pageserver HTTP API at Lsn. We would need:
- skipping `zenith.signal` (not part of a postgres fullbackup)
- deterministic order for listings

Then just write the tar to a Sha256 sink. At least the `test_import.py` would benefit from this, and I think it would be a primitive we could easily find other uses.

Reply:
> I would prefer to leave it as a follow-up perhaps..

+1

> It would be nicer to get a fullbackup hash from pageserver HTTP API at Lsn. We would need:

Agree generally, but I don't think we should invest time in fullbackup right now.

Reply:
https://github.com/neondatabase/neon/issues/7715